docs/testing.md

```diff
@@ -169,8 +162,9 @@ which checks the JavaScript against the rules configured in `.eslintrc`.
 for detailed rule descriptions.
 
 There are a number of options to the command:
- - Use `gulp lint:build` to only lint the build scripts.
- - Use `gulp lint:test` to only lint the test scripts.
- - Use `gulp lint:scripts` to only lint the project source scripts.
- - Add the `--fix` flag (like `gulp lint --fix`) to auto-fix
+
+ - `gulp lint:build`: Lint only the gulp build scripts.
+ - `gulp lint:test`: Lint only the test scripts.
+ - `gulp lint:scripts`: Lint only the project source scripts.
+ - `--fix`: Add this flag (like `gulp lint --fix`) to auto-fix
```

Comment:
That's true, I guess it doesn't hurt to document that it works. The idea of just doing `gulp lint:build` is so the build warnings/errors can be seen alone without having to scroll through all the `gulp lint:scripts` warnings if that's the area you're working in. Makes less sense for autofixing, but yeah maybe someone wants a PR that's isolated in only the tests or whatever.

Fixed in https://github.com/cfpb/cfgov-refresh/pull/2364/commits/cf0a0e4900fc40756d64821ee20d8af4a95e4395